requirements/env_common.yml

```diff
@@ -1,6 +1,7 @@
 name: deepchem
 channels:
   - conda-forge
+  - bioconda
```

Comment:
This `bioconda` can probably be removed since `hhsuite` is only included in `env_mac.yml` and `env_ubuntu.yml` and having `bioconda` there will be sufficient.